README.md

```diff
@@ -1,24 +1,89 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget-App
 
-Things you may want to cover:
+## Description
 
-* Ruby version
+In this project, I created a budget website in which users can track their finances and what they're being spent on. 
 
-* System dependencies
+## Screenshots
 
-* Configuration
+![image](https://user-images.githubusercontent.com/74814780/159991267-1dcae1b2-f4a4-4837-be7a-dc4877926c5b.png)
 
-* Database creation
+![image](https://user-images.githubusercontent.com/74814780/159991298-b0889634-e711-4676-89e0-fc7f0d271367.png)
 
-* Database initialization
+## Built with
 
-* How to run the test suite
+- Rails
 
-* Services (job queues, cache servers, search engines, etc.)
+- HTML
 
-* Deployment instructions
+- CSS
 
-* ...
+- Linters(Rubocop and Stylelint)
+
+## Live Link
+[Demo Link](http://127.0.0.1:3000/users/sign_up)
```

Comment:
Kindly ensure to deploy your app and add the demo link accordingly:
- [ ] The application should be deployed using Heroku, Netlify or GitHub Pages.
- [ ] The project is deployed properly, the home and detail page works as expected.
- [ ] A link to the deployed project is present in the README.